lib/main.dart

```diff
@@ -23,6 +24,7 @@ class FvmApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
+    checkForUpdates();
```

Comment:
Could we move this to a `useEffect` hook on the appShell?

Reply:
If I do that then every time the appShell rebuilds the popup will re-appear even if you've dismissed it.